src/dashboard/pages/index/controller.py

```diff
@@ -94,17 +112,17 @@ def display_error(title_err: bool, desc_err: bool) -> tuple[str, str, str, str]:
         tuple[str, str, str , str]: title, desc, title placeholder,
                                     desc placeholder
     """
-    title = [input_css, ""]
-    desc = [input_css, ""]
+    title_css = deepcopy(input_css)
+    title_placeholder = ""
+    desc_css = f"{deepcopy(input_css)} h-[17rem] "
```

Comment:
deepcopy should not be used here as strings are immutable and are already "copied" when assigned to variables or when manipulated.